src/namicorn.ts

```diff
@@ -50,7 +50,7 @@ class Namicorn {
     if (this.blockchain) {
       return await this.resolveUsingBlockchain(domain);
     } else {
-      const response = await fetch(`${this.api}/${domain}`);
+      const response = await window.fetch(`${this.api}/${domain}`);
```

Comment:
There is no `window` in node.js env.  You need to ensure it is working in both environments: browser and node.

Reply:
made a check for enviroments